.github/workflows/pull-request.yml

```
@@ -295,7 +295,118 @@ jobs:
+      REDIS_HOST: redis-stack-server
+      REDIS_PORT: 6379
+      ACCESS_TOKEN_SECRET: ${{ github.event.repository.name }}_access_${{ github.sha }}
+      REFRESH_TOKEN_SECRET: ${{ github.event.repository.name }}_refresh_${{ github.sha }}
+      LAST_RESORT_SUPERADMIN_EMAIL: "abc@gmail.com"
+      COLORIZE_LOGS: "true"
+      LOG_LEVEL: "info"
+      RECAPTCHA_SITE_KEY: ${{secrets.RECAPTCHA_SITE_KEY}}
+      RECAPTCHA_SECRET_KEY: ${{secrets.RECAPTCHA_SECRET_KEY}}
+      MAIL_USERNAME: ${{secrets.MAIL_USERNAME}}
+      MAIL_PASSWORD: ${{secrets.MAIL_PASSWORD}}
+             
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)